### ISSUE TITLE
Arduino support for ELRS Telemetry

### ISSUE THREAD

**Issue by CarlouT** [tbs-fpv/freedomtx#27]
I know this is a big request but since telemetry devices for Arduino is still limited at the moment because it has to be added to the hardware and manufactuers has only supported VBat, GPS and sometimes a barometer, I would like to request for the following:
 Is it possible to use an Arduino board to collect the data of sensors (i.e. battery voltage and current, GPS, Barometer, RPM etc.) and then send the telemetry data via the Tx of the Arduino connected to the Rx of the receiver. If possible, kindly provide a library that can be used to read/process and transmit the data.
![Proposal](https://github.com/tbs-fpv/freedomtx/assets/107160072/26619931-5894-40d1-8961-913d2b5c28d5)

**Comment by kamik2** [comment]
as the quickest solution, you can get the matek boards which provide voltage, current, baro (only baro altitude,  but nor rise/sink rate for vario), and PWM outputs

**Comment by CarlouT** [comment]
> as the quickest solution, you can get the matek boards which provide voltage, current, baro (only baro altitude, but nor rise/sink rate for vario), and PWM outputs

Yup I have the MATEKSYS EXPRESSLRS/ELRS 2.4GHZ Vario Receiver ELRS-R24-V ESP8285 SX1281. But it look like this board was designed for gliders. I need a receiver for a petrol plane henc I m ade this request if possible. But thanks for your advise.

**Comment by kamik2** [comment]
sounds like you also need a fuel sensor.
maybe the [openXsensor](https://github.com/mstrens/oXs_on_RP2040) project is what you are looking for.

Not sure if the CRSF implementation is already done or under development

-edit- I linked to the newer version which supports crsf

**Comment by CarlouT** [comment]
Wow, I've read the functionality of openXsensor. It's packed with features. Thanks for sharing. My only problem is its beyond my skill. I will try to go through the youtube videos and see if I can follow. Thanks.
